### ISSUE TITLE
Hyperparameter settings

### ISSUE THREAD

**Issue by JungHoyoun** [ryanxhr/POR#2]
Hi. Thanks for your excellent work. 

I keep trying to reproduce your results in Mujoco Task with the code here itself.

But I have some trouble getting the exact figure as you.

Hyperparameters are set as follows:

![image](https://user-images.githubusercontent.com/23326843/206621284-907bf34c-cce4-44c6-9e9d-689953bb4cd0.png)

and I set no argument for --g_v, --no_e_weight in order to use residuals of V as weights to learn guide, and execute policy. 

And when it comes to alpha

you mentioned in the Open Review that you set it to 0.5 in Mujoco-task, so I set it too.

![image](https://user-images.githubusercontent.com/23326843/206623470-1e7e1fa0-5780-4349-a5ec-c1b7f1eaf2ff.png)
But I eventually got these plots (seed 0~5) totally different from your paper. It was so unstable and didn't reach that reported point.

![image](https://user-images.githubusercontent.com/23326843/206623322-126848e0-dfb0-46c1-bc03-c7ecc747ac49.png)

I tried other alphas (20 to 0.01) and switched g_v and no_weight values but I failed.

Could you let me know in detail how I can get the same results as you?

**Comment by ryanxhr** [comment]
Hi, really sorry for the late reply

We apologize for the bug in our code from the previous version. We have updated the code.

Note that there is some mismatch between our provided code and the code that we used to produce the paper results. The copyright of our original code belongs to the previous company I worked for. Unfortunately, we didn't have permission to open-source it and the current github version is our re-implementation. Although we try to reimplement it as similar to the original version as possible, there is still one dataset, "hopper-medium-v2" datasest, where our re-implementation version achieves a 78.6 score, not able to reproduce 98.2 in the white paper. Other scores almost match the results in our paper (may have some minor difference due to random seeds)

Note that some configurations and experimental details are different from the original code (e.g., the value of $\alpha$, learning rate, and so on). We have updated our paper in the arxiv to reflect that and the mismatch results in the hopper-medium-v2 dataset (lower it from 98.2 to 78.6). If you want to compare POR with your work, you can refer to this paper version.

Let me know if you have any further questions!
